### PR TITLE
Change: Update the README file template

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
...
+on GitHub.
+
+## Maintainer
+
+This project is maintained by [Greenbone Networks GmbH](https://www.greenbone.net/).
+
+## Contributing
+
+Your contributions are highly appreciated. Please
+[create a pull request](https://github.com/greenbone/python-gvm/pulls) on GitHub.
```
Comment (bjoernricks) [inline README.md:47]:
```suggestion
[create a pull request](https://github.com/greenbone/<project>/pulls) on GitHub.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+For bigger changes, please discuss it first in the
+[issues](https://github.com/greenbone/python-gvm/issues).
```
Comment (bjoernricks) [inline README.md:49]:
```suggestion
[issues](https://github.com/greenbone/<project>/issues).
```

**Review by bjoernricks — COMMENTED** [review]
We could also change this into a jinja2 template and render a fitting README while creating a new repo.

**Comment by y0urself** [comment]
> We could also change this into a jinja2 template and render a fitting README while creating a new repo.

I have nearly no experience in jinja2 ...

**Pull request closed without merging**